apps/site/src/components/Sticker/BaseSticker.tsx

```diff
@@ -61,14 +67,33 @@ export default function Sticker({
 		: {};
 
 	return (
-		<motion.img
-			className={styles.stickerContainer}
-			animate={animateProps}
-			src={imageSrc}
-			alt={alt}
-			height={height}
-			width={width}
-			{...drag}
-		/>
+		<div
+			className={styles.stickerPositionContainer}
+			style={{
+				flexDirection:
+					position === "top-left" || position === "bottom-left"
+						? "row-reverse"
+						: "row",
+				alignItems:
+					position === "bottom-left" || position === "bottom-right"
+						? "flex-end"
+						: "flex-start",
+			}}
+		>
+			{children}
```

Comment:
Just thought of this, but having the BaseSticker structure be the `children` components and the motion.img sticker wrapped together in a container div might not scale well with multiple stickers, since it requires some `*Sticker.tsx` to wrap an existing component. So we would have to continually wrap Stickers onto each other. Let me know what you think on this.

Reply:
> So we would have to continually wrap Stickers onto each other.

I'm not sure what you mean by this part. The idea of wrapping a component in the sticker component is to be able to position the sticker on one of the 4 corners.

Reply:
For example, there are 5 stickers near the top of the Home Page. For them to have responsive position, they have to be "anchored" to existing components like you did with by wrapping the HeartSticker around the Apply Button.

<img width="679" alt="Screen Shot 2023-10-22 at 3 59 06 AM" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/66880934/644f6611-468e-4fc3-8c17-85155df0a640">

Reply:
I guess my overall questions is how would we achieve including multiple stickers with this design? and furthermore, is using the flex-box corner strategy "general-use enough" to scale for other stickers?

Reply:
Ah I see. To clarify, it would be difficult to anchor multiple stickers to a single element.

Reply:
Ah gotcha, we could have this be specific for the HeartSticker, if need be and if a general use solution for all stickers doesn't exist. And then make them all specific depending on the sticker.

Reply:
Sorry, that was meant to be a question.

> To clarify, it would be difficult to anchor multiple stickers to a single element.

I wanted to confirm this was the issue you were getting at.

Reply:
Yea that's what I was getting at.

Reply:
Are we good with this @tyleryy @alexanderl19 ? If so, we should merge. It looks good from my end!

Reply:
I'm ok with either merging this now or continuing to working on it. This does get rid of the dead zone issue on prod I believe so merging now would be good. However, I think we still need more work on the sticker positioning, which could be addressed in another PR. @alexanderl19 's call.

Reply:
I really wish GitHub had a better way of showing when I've been mentioned. I'm going to rework the API, and this PR should be good to merge EOD today (Sunday).